core/src/main/java/eu/tsystems/mms/tic/testframework/utils/SourceUtils.java

```diff
@@ -51,7 +53,8 @@ private SourceUtils() {
     private static final Logger LOGGER = LoggerFactory.getLogger(SourceUtils.class);
 
     private static String sourceRoot = System.getProperty(TesterraProperties.MODULE_SOURCE_ROOT, "src");
-    private static int linePrefetch = PropertyManager.getIntProperty(TesterraProperties.SOURCE_LINES_PREFETCH, 5);
+    private static int linePrefetch = DefaultReport.Properties.SOURCE_LINES_PREFETCH.asLong().intValue();
```

Comment:
I prefer retrieving property values exactly the time you need it and do not store them in a static variable that cannot be manipulated anymore, which is the purpose of properties when you think about having global and thread local properties as well.